prime-router/src/main/kotlin/azure/SendFunction.kt

```diff
@@ -29,14 +29,27 @@ import kotlin.random.Random
 /**
  * Azure Functions with HTTP Trigger. Write to blob.
  */
-const val dataRetentionDays = 7L
```

Comment:
Unused removed

Reply:
Yes, this is a leftover.   I believe this number has been moved to the react code.